example/lib/phyllo_provider.dart

```diff
@@ -92,6 +92,9 @@ class PhylloProvider extends DefaultChangeNotifier {
       },
     );
 
+    //get supported version details
+    log('version: ${_phylloConnect.version()}');
```

Comment:
Can you print the sample output here, just for review?

Reply:
[log] version: {connect_flutter_sdk_version: 0.3.2, connect_android_sdk_version: 0.3.1, min_supported_android_version: 21, max_supported_android_version: 31, connect_ios_sdk_version: 0.3.1, min_supported_ios_version: 12.0, max_supported_ios_version: 16.0*}